### ISSUE TITLE
[REQUEST] Add disk IO meter

### ISSUE THREAD

**Issue by baybal** [aristocratos/bpytop#105]
**Is your feature request related to a problem? Please describe.**
I lack disk I/O usage meters

**Describe the solution you'd like**
Please add a meter for disk io time usage, and per-process measurement

**Comment by aristocratos** [comment]
> Please add a meter for disk io time usage

Please expand on that. Time spent reading/writing since system start as a percentage? Two extra meters for every disk would take up a lot of space which is already pretty cramped when there are multiple drives/partitions present.

> and per-process measurement

Isn't present in psutil, see https://psutil.readthedocs.io/en/latest/#psutil.Process.io_counters for what's available.

**Comment by baybal** [comment]
> Please expand on that. Time spent reading/writing since system start as a percentage?

For the system, the percentage of time the system spent in IO wait.

For the process. Percentage of system's total read/write traffic used by the process.

**Comment by baybal** [comment]
Those were the most useful features for me. They are kindof not the same, and I think having the two separate would be better. So it will be mbps for the system, and the process, and iowait time percentage for the system, and an individual process.

**Comment by aristocratos** [comment]
> For the system, the percentage of time the system spent in IO wait.

That would be a per disk percentage and not a system wide percentage, that's what I meant with it taking up a lot of space if added for every disk. But could be added and have to be toggled ON in the options in that case.

> For the process. Percentage of system's total read/write traffic used by the process.

This is already shown in the process info box, but it gets cut out if the width is too low, see picture.

![io-stats](https://user-images.githubusercontent.com/59659483/92616107-2368cc00-f2be-11ea-81c1-e0adde060eac.png)

**Comment by baybal** [comment]
Thank you, never noticed it.

But separately, is there any way to measure the percentage of time the process spends in iowait? The main utility of this is to find processes which iolock the system

**Comment by aristocratos** [comment]
> But separately, is there any way to measure the percentage of time the process spends in iowait? The main utility of this is to find processes which iolock the system

Yeah, `cpu_times()` have representation of total iowait in seconds per process (which could be calculated to percent from process run time), see https://psutil.readthedocs.io/en/latest/#psutil.Process.cpu_times

But I would be hesitant to add this to the default view of the processes since it isn't a high priority stat for most people.

**Comment by khimaros** [comment]
i would also love to see this, even if it were not enabled by default. i often find myself hopping back and forth between htop and iotop for a full picture of system utilization.
